docs/migration_guide.md

```diff
@@ -0,0 +1,115 @@
+    ```
+
+
+## Migration procedure
+
+1. [Install](install/installEverest.md) everest.
+3. Migrate backup storages from PMM to Percona Everest:
+
+    - Log in to the PMM UI and navigate to the **Backup > Storage Locations** section. The **Storage Location** page opens. You can see the storage locations configured on this page.
+    - Click the downward arrow to see the details of the storage location that are configured.
+    - Log in to Percona Everest and go to the **[Backup Storages](use/backupRestore.md#prepare-storage-location)** section on the UI.
+    - Copy-paste the values from the **details** on the configured storage locations in PMM and paste it to **[Add backup storage](use/backupRestore.md#prepare-storage-location)** page in Percona Everest.
+
+4. Migrate secrets for database clusters from one Kubernetes cluster to a new cluster using the pattern `everest-secrets-dbclusterName`. 
+
+    Example:
+
+    ```sh
+    export DBNAME=mysql-btknhj
+    export DBTYPE=pxc
+    ```
+
+    !!! note alert alert-primary "Note"
+        The DBTYPE can be `pxc` or `psmdb` depending on whether it is a MySQL or MongoDB database.
+
+    Get the secret from the kubernetes cluster registered in PMM:
+
+    ```sh
+    kubectl get secret "dbaas-$DBNAME-$DBTYPE-secrets" -o yaml | sed "s/name: dbaas-$DBNAME-.*-
+    secrets/name: everest-secrets-$DBNAME/" | sed "s/namespace: default/namespace: percona-everest/" > 
+    secret.yaml
+    ```
+
+    Apply the secret in the kubernetes cluster registered in Everest:
+
+    ```sh
+    kubectl apply -f secret.yaml
+    ```
+
+
+    !!! note alert alert-primary "Note"
+        Keep the naming consistent across the two Kubernetes clusters.
+
+5. [Create a new database cluster](use/db_provision.md) using the webUI. Run through the creation wizard to select resources, set the name, and configure the backup storage and monitoring of the cluster.
+6. Restore data for your database clusters using the operator's backup and restore features.
+
+    !!! note alert alert-primary "Note"
+        - The restoration won't be available for you in the web UI, you'll need to refer to the corresponding operator's documentation for instructions on how to do this.
+
+            -  For PXC https://docs.percona.com/percona-operator-for-mysql/pxc/backups-restore.html
+            -  For PSMDB https://docs.percona.com/percona-operator-for-mongodb/backups-restore.html
+            -  For PostgreSQL https://docs.percona.com/percona-operator-for-postgresql/2.0/backups-restore.html
+
+        - Each time you run a database cluster, repeat this step.
+
+8. Disable the DBaaS feature in PMM by navigating to <i class="uil uil-cog"></i> **Configuration** → <i class="uil uil-setting"></i> **Settings** → **Advanced Settings** on the PMM UI and toggling <i class="uil uil-toggle-off"></i> the **Database as a Service (DBaaS)** option located in the **Technical preview features** section.
```

Comment:
After removing step 2 you forgot to renumber the steps accordingly, now we have 1-3-4-5... instead of 1-2-3-4...

Reply:
Done